README.md

```diff
@@ -187,7 +183,7 @@ which means you can modify it, redistribute it or use it however you like.
                                      of SIZE.
     --playlist-reverse               Download playlist videos in reverse order
     --xattr-set-filesize             Set file xattribute ytdl.filesize with
-                                     expected filesize (experimental)
+                                     expected file size (experimental)
```

Comment:
Thanks, that was not obvious.
Shall I change it back in the README, or is the [*actual* change in options.py](https://github.com/habi/youtube-dl/commit/8fbfdb3ae926c6162e06643406a7dc9080f3f454#diff-09f445e8685f0e866d7ff74c9481feec) anyways going to overwrite the README?